### ISSUE TITLE
Add option to disable autoprefixer

### ISSUE THREAD

**Issue by niksajanjic** [csstools/postcss-preset-env#87]
I would like to be able to disable autoprefixer completely. Could we enable an option of passing autoprefixer option as false. Something like this:

```js
const stagedAutoprefixer = autoprefixerOptions !== false &&  autoprefixer(Object.assign({ browsers }, autoprefixerOptions));

...

.then(
  () => stagedAutoprefixer ? stagedAutoprefixer(result.root, result) : result // or whatever needs to be returned
);
```

**Comment by jonathantneal** [comment]
Yes, that seems entirely reasonable. I was surprised the option wasn’t already available, and I almost figured the documentation just failed to mention it. If I haven’t gotten to this soon, feel free to submit a PR any time.

**Comment by jonathantneal** [comment]
@niksajanjic, out of curiosity, what is your use case for disabling autoprefixer? Are you still using a browserslist?

**Comment by niksajanjic** [comment]
Of course, no problem. I apologize up front because explanation will be long. My use case is not a very good reason to add an option to disable autoprefixer, but I still feel there should be an option to disable it. I'm trying to use `postcss-modules` along with `postcss-preset-env`. My `.postcssrc.js` looks like this:

```js
const path = require('path')

module.exports = ctx => ({
  map: ctx.options.map,
  parser: 'sugarss',
  plugins: {
    'postcss-easy-import': {
      extensions: '.sss',
      path: path.resolve('.'),
      plugins: [
        require('postcss-mixins'),
        require('postcss-preset-env')({
          autoprefixer: {
            add: false,
          },
          stage: 2,
          features: {
            'nesting-rules': true,
          },
        }),
        require('postcss-modules')({
          generateScopedName: '[name]__[local]___[hash:base64:5]',
        }),
      ],
    },
    'postcss-preset-env': {
      autoprefixer: {
        grid: true,
      },
      stage: 2,
    },
    'postcss-discard-duplicates': {},
    'postcss-browser-reporter': {},
    'postcss-reporter': {},
  },
})
```

I'm using `postcss-cli` to compile my `style.css`. Because I have multiple files scattered around the project I have 1 `main.sss` where I'm importing all the files that I need, like this:
```sss
// ---------- GLOBAL & RESETS ----------
@import 'client/style/shared/global/variables'
@import 'client/style/shared/global/mixins'
@import 'client/style/shared/global/defaults'
@import 'client/style/shared/global/globals'

// ---------- THEMES ----------
@import 'client/style/shared/themes/pikaday'

// ---------- PAGES ----------
@import 'views/pages/**/*'

// ---------- COMPONENTS ----------
@import 'views/components/**/*'

```
With `postcss-cli` I can't pass multiple input files with only 1 output. Therefore I have to use `postcss-easy-import` to load them into 1 file and then process that file through `postcss-cli`. Also, I have to put `postcss-modules` inside `postcss-easy-import` because I want every imported file to be separately processed and JSON file created for each file. If i do it outside, it will inline rules and then process them as one `main.sss` file and create 1 JSON output file. That is not good for me in cases I have same class names in different files (as they will override each other).

Now, the only explanation left is why do I have 2 `postcss-preset-env`'s. Well, with `postcss-modules` it is possible to have `:global` rules. That is great and easy but it doesn't support nesting out of the box. That is why I have to run first `postcss-preset-env` with nesting feature so I can resolve nests before I pass those rules to `:global` and it works great.

Of course, first time I did this is I put every other feature inside first `postcss-preset-env` but it didn't work because of the fact that there is some bug inside `postcss-modules`. Basically, if my autoprefixer is run before `postcss-modules` and if I have `.test::prefix` rule, that rule is going to be transformed into:
```css
.test::placeholder {}
.test::-ms-input-placeholder {}
```

After that, `postcss-module` runs and traslates that into:

```css
.file__test___63dhs::placeholder {}
.test::-ms-input-placeholder {}
.file__test___63dhs::-ms-input-placeholder
```

It creates this global variable for some reason that I don't want to have. And it's even worse if I intended it to be `:global`:
```css
.test::placeholder {}
:global .test::-ms-input-placeholder {}
.test::-ms-input-placeholder
```

Because of this bug I have to turn of autoprefixer before and turn it on back again after `postcss-modules` is done.

I'll take a look over the weekend and possibly make a PR for this feature.

**Comment by jonathantneal** [comment]
Could you also use `insertAfter`? https://github.com/csstools/postcss-preset-env#insertbefore--insertafter

```js
require('postcss-preset-env')({
  insertAfter: {
    'nesting-rules': require('postcss-modules')
  }
})
```

**Comment by niksajanjic** [comment]
> Could you also use `insertAfter`? https://github.com/csstools/postcss-preset-env#insertbefore--insertafter
> 
> ```js
> require('postcss-preset-env')({
>   insertAfter: {
>     'nesting-rules': require('postcss-modules')({
>       generateScopedName: '[name]__[local]___[hash:base64:5]',
>     })
>   }
> })
> ```

Although this is a great idea it doesn't work in my code example. Basically, it will loop forever and constantly keep importing the first file. I'm pretty sure this issue has nothing to do with `postcss-preset-env` because I tried it with other plugins and they work as expected. Tried to debug it and figure out the issue but didn't get too far.

Also, while debugging, I realized that it is much slower to run each file individually through `postcss-preset-env` so that option wouldn't be a solution for me. It is better to run `postcss-preset-env` after all files have been imported and run through `postcss-modules` and then run a file as a whole as it's much less expensive performance wise. My config does look awful, but it does the job.

On a bright side, thanks for adding the option to disable autoprefixer. It works great.
